g4f/Provider/selenium/PerplexityAi.py

```diff
@@ -6,10 +6,10 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.keys import Keys
 
-from ..typing import CreateResult, Messages
-from .base_provider import AbstractProvider
-from .helper import format_prompt
-from ..webdriver import WebDriver, WebDriverSession
+from ...typing import CreateResult, Messages
```

Comment:
Consider organizing import statements in alphabetical order for better readability.